presto-docs/src/main/sphinx/functions/regexp.rst

```diff
@@ -115,3 +115,28 @@ with a few notable exceptions:
     array. Trailing empty strings are preserved::
 
         SELECT regexp_split('1a 2b 14m', '\s*[a-z]+\s*'); -- [1, 2, 14, ]
+
+.. function:: regexp_count(string, pattern) -> integer
```

Comment:
Nit: we normally alphabetize functions within a documentation section

Reply:
This could be done today as `cardinality(regexp_extract_all(...))`, but this version is more convenient and easier for the reader to understand.